src/main/java/cz/cvut/kbss/study/persistence/dao/PatientRecordDao.java

```diff
@@ -284,23 +284,26 @@ private void setQueryParameters(Query query, Map<String, Object> queryParams) {
                            URI.create(Vocabulary.s_p_was_treated_at))
              .setParameter("hasKey", URI.create(Vocabulary.s_p_key))
              .setParameter("hasCreatedDate", URI.create(Vocabulary.s_p_created))
-             .setParameter("hasLastModified", URI.create(Vocabulary.s_p_modified));
+             .setParameter("hasLastModified", URI.create(Vocabulary.s_p_modified))
+             .setParameter("institutionGraph", URI.create(Vocabulary.s_c_institution + "s"));
```

Comment:
@kostobog Not sure but i think this is not good.

Reply:
That is i think why tests are failing.

Reply:
Is it really needed to query concrete graph? Can't you query default graph ?

Reply:
Yes this is a part of the reason. Tests are failing because tests create data in the default context but the optimization looks for them in named contexts. 
To fix the tests I will revert this optimization for JSON export.